airbyte-integrations/connectors/source-zendesk-chat/source_zendesk_chat/manifest.yaml

```diff
@@ -316,3 +329,8 @@ check:
   type: CheckStream
   stream_names:
     - routing_settings
+
+concurrency_level:
+  type: ConcurrencyLevel
+  default_concurrency: 10
```

Comment:
The limits from Zendesk Chat seem somewhat low i.e. [200 requests per minute for the team plan](https://developer.zendesk.com/api-reference/introduction/rate-limits/#zendesk-support-plan-limits). I fear we might hit this limit with 10 concurrent threads. Should we set API budget at least?

Reply:
👍  added api budget, `200/60 = 3.3` -> we can set default concurrency to `4` later if `10` is too high.